tests/tests/test_data_loader.py

```diff
@@ -20,49 +22,28 @@ def test_oversample(self):
         ]
 
     def test_batch_structure(self):
-        ds = TextMelDataset(
+        ds = Data(
             "tests/fixtures/val.txt",
-            ["english_cleaners"],
-            0.0,
-            80,
-            22050,
-            0,
-            8000,
-            1024,
-            256,
-            padding=None,
-            win_length=1024,
             debug=True,
             debug_dataset_size=12,
             symbol_set="default",
         )
         assert len(ds) == 1
-        collate_fn = TextMelCollate()
+        collate_fn = Collate()
         dl = DataLoader(ds, 12, collate_fn=collate_fn)
         for i, batch in enumerate(dl):
-            assert len(batch) == 8
+            assert len(batch) == 9
```

Comment:
This seems like an interesting thing to change, why did it change?

Reply:
f0